pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md

```diff
@@ -0,0 +1,87 @@
+
+![changement d'offre](images/manager-changeOffer-2024.png){.thumbnail}
+
+La première colonne du tableau récapitule votre offre actuelle (son nom et son prix). Les autres colonnes concernent les offres auxquelles vous pouvez souscrire.
+Dans la deuxième ligne du tableau, vous pouvez, si vous le souhaitez, commander un boitier OverTheBox v3 en plus de votre changement d'offres.
+
+Une fois votre choix effectué, cliquez sur le bouton `Sélectionner`{.action} sous la colonne correspondant à l'offre que vous souhaitez souscrire.
+
+![selection d'offre](images/manager-selectOffer-2024.png){.thumbnail}
+
+### Sélectionner une addresse de livraison
+
+> [!primary]
+> Si vous avez choisi l'option `Je ne souhaite pas de boitier`{.action} vous pouvez passer cette section.
+
+Si vous avez choisi de commander un nouveau boitier avec votre changement d'offre, vous devez choisir une adresse de livraison.
+Par défaut l'adresse est celle de votre compte OVHcloud, vous pouvez modifier cette adresse :
+
+* Cliquer sur le bouton `+`{.action} permet de renseigner une nouvelle adresse.
+* Cliquer sur le bouton `loupe`{.action} permet de sélectionner une adresse précédemment renseignée.
+
+Une fois l'adresse saisie, cliquez sur le bouton `Sélectionner le contact`{.action}
+
+> [!primary]
+> Les frais de livraison sont inclus dans le prix du boitier
+
+![selection d'adresse de livraison](images/manager-selectAddress-2024.png){.thumbnail}
+
+### Valider le changement d'offre
+
+La page `Résumé du changement d'offre` résume la tarification mensuelle de la nouvelle offre, ainsi que le cout du boitier (si l'option a été sélectionnée).
+
+Validez les contrats afférents, puis cliquez sur le bouton `Valider le changement d'offre`{.action}
+
+![validation de la migration](images/manager-confirmMigration-2024.png){.thumbnail}
+
+### Payer le bon de commande
+
+> [!primary]
+> Une fois le bon de commande payé le changement d'offre est immédiat. Un remboursement au prorata de la facturation du mois en cours est établi.
+
+Pour finaliser le changement d'offre, ainsi que la livraison du boitier, vous devez payer le bon de commande généré lors de la confirmation de la migration.
+Pour ce faire cliquez sur le lien contenu dans la phrase `Pour finaliser la migration de votre offre, vous devez payer le bon de commande.`{.action}.
+
+> [!primary]
+> Pour plus d'informations sur le paiement d'un bon de commande, reportez-vous sur le guide [Gérer mes commandes OVHcloud](https://help.ovhcloud.com/csm/fr-billing-managing-ovh-orders?id=kb_article_view&sysparm_article=KB0042874)
+
+![paiement](images/manager-payment-2024.png){.thumbnail}
+
+## Aller plus loin
+
+N'hésitez pas à échanger avec notre communauté d'utilisateurs sur vos produits Télécom sur notre site [OVHcloud Community](https://community.ovh.com/c/telecom)
+
+Plus d'informations sur [les offres OverTheBox](https://www.ovhtelecom.fr/overthebox/)
```

Comment:
Migration is available but some issue on website side for new offers pages, will update once it prod